uber/site_sections/registration.py

```diff
@@ -153,17 +155,15 @@ def delete(self, session, id, return_to = 'index?'):
                 session.delete_from_group(attendee, attendee.group)
                 message = 'Unassigned badge removed.'
             else:
-                #session.add(Attendee(**{attr: getattr(attendee, attr) for attr in [
-                #    'group', 'registered', 'badge_type', 'badge_num', 'paid', 'amount_paid', 'amount_extra'
-                #]}))
+                session.add(Attendee(**{attr: getattr(attendee, attr) for attr in [
+                    'group', 'registered', 'badge_type', 'badge_num', 'paid', 'amount_paid', 'amount_extra'
+                ]}))
                 session.assign_badges(attendee.group, attendee.group.badges + 1, attendee.badge_type)
-                Tracking.track(INVALIDATED, attendee)
-                #session.delete_from_group(attendee, attendee.group)
+                session.delete_from_group(attendee, attendee.group)
```

Comment:
Be sure to keep us in the loop on how you plan to do this! This was an "emergency patch" so I definitely would like it replaced with something better.